File: 241026/배수의 개수/number-of-multipliers.cpp
#include <iostream>
using namespace std;

int main() {
    int cntA = 0, cntB = 0;
    int a;

    for (int i = 1; i <= 10; i++) {
        cin >> a;

        if (a % 3 == 0) {
            cntA++;
        }

        if (a % 5 == 0) {
            cntB++;
        }
    }

    cout << cntA << " " << cntB;
    return 0;
}

File: 241026/배수의 개수/number-of-multipliers.js
// 변수 선언 및 입력
const fs = require("fs");
let input = fs.readFileSync(0).toString().trim().split("\n");
let cntA = 0, cntB = 0;

for (let i = 0; i < 10; i++) {
    let x = Number(input[i]);
    if (x % 3 === 0) {
        cntA++;
    }

    if (x % 5 === 0) {
        cntB++;
    }
}

// 출력
console.log(cntA, cntB);